website/content/commands/server-restore.mdx

```diff
@@ -15,7 +15,20 @@ Stage a snapshot on the current server for data restoration
 
 ## Usage
 
-Usage: `waypoint server restore [options]`
+Usage: `waypoint server restore [-exit] [<filename>]`
+
+    Stage a backup snapshot within the current server. The data in the snapshot is not restored
+    immediately, but rather staged such that on the next server start, it will be restored.
+
+    If -exit is passed, the server process will exit after staging the data. This allows a process
+    monitor to restart the server, where it will see the staged snapshot and restore the data.
+
+    If -exit is not passed, an operator must restart the server manually to finish the restoration
+    process.
+
+    The argument should be to a file written previously by 'waypoint server snapshot'.
+    If no name is specified and standard input is not a terminal, the backup will read from
+    standard input. Using a name of '-' will force reading from standard input.
```

Comment:
Same issue here - this whole thing is in a code block:

<img width="1039" alt="Screen Shot 2021-08-10 at 9 48 54 AM" src="https://user-images.githubusercontent.com/8404559/128878670-157716ad-3860-40e2-b1e9-e6407cae4322.png">